### ISSUE TITLE
maybe pass pug flags down to the template render call?

### ISSUE THREAD

**Issue by misterbrownlee** [topcoat/topdoc#49]
I'm wanting my Hyper Text Markup Language to have some`--pretty` without having to write my own template js or hack the default template in node_modules.  However, maybe nobody else wants this feature?  I think I'll take a poke at it anyway, unless someone stops me.  

Ultimately the opts have get passed down to [topdoc-default-template](https://github.com/topcoat/topdoc-default-template), but having them get parsed out of .topdocrc` would be the right thing, which is handled here.  I don't see other flags that would make much sense, and I don't know if mucking with an array of flags is worth it... not for now anyway. At the least I can start with the thing I need (`setPugPretty: true` or something), and folks can pile on if they want other flags?  

It could be simple I think... `if (config.setPugPretty) ...` and just hand off and extra arg to the renderer call [here](https://github.com/topcoat/topdoc-default-template/blob/master/src/index.js#L51-54).  It might make a little ugly if else, but I could maybe build the call params prior to the call to clean it up a little?

WDYT y'alls?

**Comment by GarthDB** [comment]
We could do a whole pug option object that is passed as an object called `pugOptions`

**Comment by misterbrownlee** [comment]
Yes.  I wasn't sure how fancy we wanted to get and ultimately I had to write my own extension of the default template and can pass whatever I want myself.  Your call, if you are keen I'm happy to tinker on this and make a pull request.
